src/helpers/library.ts

```diff
@@ -195,11 +192,22 @@ const launch = (appName: string, t: TFunction<'gamepage'>, handleGameStatus: (ga
 const updateGame = (appName: string): Promise<void> =>
   ipcRenderer.invoke('updateGame', appName)
 
+// Todo: Get Back to update all games
```

Comment:
Is this queue based? Otherwise exposing an endpoint for the updateAll function already present on the backend would be smarter imo.

Reply:
the problem is the backend part is still not connected with the frontend, so if we update a game there, there will be no feedback.
But, you have a point. I tried to implement this using a queue system on GlobalState, so I was checking first if there was any game on the libraryStatus with a status different from playing. So if we have a game installing, updating or repairing, the update would wait for it to finish.

But I could not test it enough since during this all my games got updated :joy: 

I think we might then use the backend implementation once you have the status and the communication going on back and forth.

But can I keep this in case we decide to call this on FE?